docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/operate-first/opf-toolbox:v0.8.0
+
+# Upgrade the system
+RUN dnf -y upgrade
+
+# Create the destination directory
+RUN mkdir -p /scripts
```

Comment:
Get rid of this `RUN` and move the `WORKDIR` here (`WORKDIR` creates the target directory if it doesn't exist).